### PR TITLE
bundler endpoint is now secret

### PR DESCRIPTION
**Pull request by NikolayRn**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Refactor or a modification (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by AliakseiMalyshau** [comment]
@NikolayRn hey. Can you explain why this env variable turned into secret?

File: k8s/helm/production.yaml
Comment (AliakseiMalyshau) [inline k8s/helm/production.yaml:1]: @NikolayRn please delete the `bundler_api_prd_url` from configMap dict for both environments.

**Review by AliakseiMalyshau — CHANGES_REQUESTED** [review]

File: k8s/helm/production.yaml
Comment (NikolayRn) [reply k8s/helm/production.yaml:1]: Done @AliakseiMalyshau

**Review by AliakseiMalyshau — APPROVED** [review]
LGTM. Secret was created only for `staging` for now. Please let me know when `staging` will be merged to the `master` and I'll create it for `production` environment.

**Pull request merged**